core/terraform/resource_processor/vmss_porter/variables.tf

```diff
@@ -97,3 +97,13 @@ variable "kv_encryption_key_name" {
   type        = string
   description = "Name of Key Vault Encryption Key (only used if enable_cmk_encryption is true)"
 }
+
+variable "enable_malware_scanning" {
+  type        = bool
+  description = "If False, Airlock requests will skip the malware scanning stage"
+}
+
+variable "scan_result_topic_name" {
+  type        = string
```

Comment:
Is there any harm in setting a default for this? Presumably, if scanning isn't enabled it would make no difference.

Reply:
The name of the topic gets generated by `locals.tf` in core so I needed to output the generated name and pass it through to the workspace so we can use the same topic. The function that sits on the end of it already had code to cope with core or workspace so it just needed hooking up.